cli/src/semgrep/commands/scan.py

```diff
@@ -329,7 +329,12 @@ def convert(
         is_flag=True,
         hidden=True,
     ),
-    optgroup.option("--allow-untrusted-validators", is_flag=True, hidden=True),
+    optgroup.option(
+        "--allow-custom-validators",
```

Comment:
@leifdreizler / @kopecs should we still keep `--allow_untrusted_validators` around but nudge users to use `--allow-custom-validators` and eventually remove it? Not sure if we have many people using this flag to begin with.

Reply:
It is hidden and undocumented. Just change it imo. Likewise, I don't see a need for a changelog entry for some things (e.g., removal of `--beta-testing-secrets`).

Reply:
Added changelog commit here: [f54e5b6](https://github.com/semgrep/semgrep/pull/9987/commits/f54e5b6224b66e5251c504bff532f0417e8de806)

I'll drop the historical secrets and allow_untrusted_validators changelog entries since their hidden flags?

Reply:
Sounds good.

Reply:
Done in e61280a96354d16cd15c3ca81e59410ec679d95e